agents/community/open-deep-research-agent/beeai-provider.yaml

```diff
@@ -0,0 +1,24 @@
+manifestVersion: 1
+driver: python
+pythonVersion: "3.13"
+package: "file:///home/paulschw/i-am-bee/beeai/agents/community/open-deep-research-agent/"
+serverType: http
+mcpTransport: sse
+mcpEndpoint: /sse
+command: ["open-deep-research-agent"]
+env:
+- name: LITELLM_MODEL_ID 
+  required: false
+  description: "Model specifier in the LiteLLM format, e.g. 'groq/deepseek-r1-distill-llama-70b'. Do not forget to set the corresponding API key -- see docs: https://docs.litellm.ai/docs/providers."
+- name: LITELLM_API_BASE
+  required: false
+  description: "Base API for model"
+- name: OPENAI_API_KEY
+  required: false
+  description: "API key for OpenAI services"
+- name: HF_TOKEN
+  required: true
+  description: "API key for Hugging Face"
+- name: SERPER_API_KEY
```

Comment:
Could this be unified with Autogen which also uses Google Search? It uses `GOOGLE_API_KEY` and `GOOGLE_SEARCH_ENGINE_ID`. I know that Serper is a different provider, but it looks like the tool could support multiple providers.

Reply:
Maybe... I haven't really dug into how this agent is using tools, but I can see what's possible.

Reply:
Actually, now that I think about it, I wouldn't mess with the tools here. Agents contributed by the community will all have their own way of doing things (and trying to make updates ourselves will be non-pleasant to maintain in the future...)